api/swagger-spec/v1.json

```diff
@@ -13193,7 +13193,7 @@
       "type": "versioned.Event",
       "method": "GET",
       "summary": "watch individual changes to a list of Secret",
-      "nickname": "watchNamespacedSecretList",
+      "nickname": "watchNamespacedSecretListForAllNamespaces",
```

Comment:
If it is important to keep the Namespaced at the beginning, maybe the second version could be watchAllNamespacesSecretList? That would keep the length down a little bit, although it's a bit less natural english.